app/assets/stylesheets/semantic_style.scss

```diff
@@ -89,44 +91,22 @@ nav {
       margin-bottom: 0;
     }
 
-    li,
-    li a,
-    li span {
-      font-weight: 800;
-      //letter-spacing: 0.025em;
-      color: $text-color;
-      text-decoration: none;
-      border-radius: 6px;
-      // padding: 0.5em 1em 0.5em 1em;
-      -moz-transition: background-color .25s ease-in-out;
-      -webkit-transition: background-color .25s ease-in-out;
-      -o-transition: background-color .25s ease-in-out;
-      -ms-transition: background-color .25s ease-in-out;
-      transition: background-color .25s ease-in-out;
-    }
-
-    li > ul {
-      display: none;
-    }
-
-    li:hover a,
-    li:hover span {
-      background: $nav-button-bkg;
-    }
-
+    // TODO: either user this .active class or delete it
     li.active a,
     li.active span {
       background: $nav-button-bkg;
     }
 
-    li.current a {
+    li.link_button a {
       background: $nav-button-bkg;
       color: $text-color;
       padding: .2rem .4rem;
     }
 
     @media (#{$bp-larger-than-mobile}) { // 400px
```

Comment:
This is an allowed usage according to SASS docs and Skeleton uses it.  But I have changed it in my code because I agree that using SASS variable expansion is preferable.